src/terminal/Line_test.cpp

```diff
@@ -36,6 +36,57 @@ TEST_CASE("Line.BufferFragment", "[Line]")
     CHECK(fragment.view() == externalView);
 }
 
+TEST_CASE("Line.resize", "[Line]")
+{
+    auto constexpr DisplayWidth = ColumnCount(4);
+    auto text = "abcd"sv;
+    auto pool = BufferObjectPool(32);
+    auto bufferObject = pool.allocateBufferObject();
+    bufferObject->writeAtEnd(text);
+
+    auto const bufferFragment = bufferObject->ref(0, 4);
+
+    auto sgr = GraphicsAttributes {};
+    auto const trivial =
+        TrivialLineBuffer { DisplayWidth, sgr, sgr, HyperlinkId {}, DisplayWidth, bufferFragment };
+    CHECK(trivial.text.view() == string_view(text.data()));
+    auto line_trivial = Line<Cell>(LineFlags::None, trivial);
+    CHECK(line_trivial.isTrivialBuffer());
+
+    line_trivial.resize(ColumnCount(10));
+    CHECK(line_trivial.isTrivialBuffer());
+
+    line_trivial.resize(ColumnCount(5));
+    CHECK(line_trivial.isTrivialBuffer());
+
+    line_trivial.resize(ColumnCount(3));
+    CHECK(line_trivial.isTrivialBuffer());
+}
+
+TEST_CASE("Line.reflow", "[Line]")
+{
+    auto constexpr DisplayWidth = ColumnCount(4);
+    auto text = "abcd"sv;
+    auto pool = BufferObjectPool(32);
+    auto bufferObject = pool.allocateBufferObject();
+    bufferObject->writeAtEnd(text);
+
+    auto const bufferFragment = bufferObject->ref(0, 4);
+
+    auto sgr = GraphicsAttributes {};
```

Comment:
```suggestion
    auto const sgr = GraphicsAttributes {};
```
Now I am questioning myself. You use `const` everywhere else by default, which is good, except for `sgr`. What am I missing? 🤔

Reply:
You are not missing anything, that was me who missed it, I just copy pasted it from another unit test where some GraphicsAttributes were used